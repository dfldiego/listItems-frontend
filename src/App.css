:root {
  /* Colors */
  --yellow: #ffe600;
  --black: #333333;
  --grey-moderate: #666666;
  --grey: #999999;
  --white-moderate: #eeeeee;
  --white: #ffffff;
  --blue: #3483fa;
  --font-roboto: "Roboto", sans-serif;
}

html {
  box-sizing: border-box;
  font-size: 62.5%;
  margin: 0;
  padding: 0;
}

*,
*:before,
*:after {
  box-sizing: inherit;
}

body {
  font-size: 1.8rem;
  width: 100%;
  font-family: var(--font-roboto);
  background-color: var(--white-moderate);
}

p {
  margin: 0;
  padding: 0;
}

.fontSize24 {
  font-size: 2.4rem;
}
.fontSize18 {
  font-size: 1.8rem;
}
.fontSize14 {
  font-size: 1.4rem;
}
.fontSize12 {
  font-size: 1.2rem;
}

.mt16 {
  margin-top: 1.6rem;
}

.mx16 {
  margin: 0 1.6rem;
}

.py32 {
  padding: 3.2rem 0;
}

.px32 {
  padding: 0 3.2rem;
}

.pr1 {
  padding-right: 1rem;
}

.container {
  width: 100%;
  margin: 0 auto;
}

@media (min-width: 768px) {
  .container {
    width: 80%;
  }
}

@media (min-width: 992px) {
  .container {
    width: 60%;
  }
}

img {
  max-width: 100%;
}

.mt5 {
  margin-top: 5rem;
}

.error {
  background-color: #ff6161;
  width: 100%;
  font-size: 10px;
  color: #fff;
  text-align: center;
  text-transform: uppercase;
  font-weight: bold;
  margin: auto auto;
  padding: 2rem;
  border-radius: 0.5rem;
  margin: 1rem 0;
}
.errorGrid {
  grid-column: 3 / 12;
  grid-row: 3 / 4;
}
.spinner {
  border: 4px solid var(--blue);
  border-left-color: transparent;
  width: 36px;
  height: 36px;
  margin: auto auto;
  padding: 2rem;
  border-radius: 3rem;
  margin: 1rem 0;
  animation: spin 1s linear infinite;
  z-index: 2;
}
@keyframes spin {
  0% {
    transform: rotate(0deg);
  }

  100% {
    transform: rotate(360deg);
  }
}
.loadingGrid {
  grid-column: 6 / 7;
  grid-row: 3 / 4;
  margin-top: 5rem;
}
.page_container {
  height: 100%;
  min-height: 100%;
  background-color: var(--white-moderate);
  display: grid;
  grid-template-columns: repeat(12, 8.33%);
}
.page_search {
  grid-column: 1 / 13;
  grid-row: 1 / 2;
  max-height: 5rem;
}
.page_breadcrumbs {
  grid-column: 2 / 12;
  grid-row: 2 / 3;
  margin: 14px 0;
}
.page_content {
  background-color: var(--white);
  grid-column: 2 / 12;
  grid-row: 3 / 10;
  display: grid;
  padding: 1.6rem;
}
.pagination_button_one {
  grid-column: 5;
  grid-row: 11 / 12;
  min-height: 5rem;
  padding: 1.6rem;
}
.pagination_button_two {
  grid-column: 8;
  grid-row: 11 / 12;
  min-height: 5rem;
  padding: 1.6rem;
}
