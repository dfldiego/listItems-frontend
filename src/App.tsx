import "./App.css";
import { createBrowserRouter, RouterProvider } from "react-router-dom";
import { appRoutes } from "./routes/appRoutes";
const router = createBrowserRouter(appRoutes);
function App() {
  return (
    <>
      <RouterProvider router={router} />
    </>
  );
}

export default App;
