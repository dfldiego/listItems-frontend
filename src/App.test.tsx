import { render } from "@testing-library/react";
import App from "./App";

describe("<App>", () => {
  test("<App /> render correctly", () => {
    const { container } = render(<App />);
    expect(container).toMatchSnapshot();
  });
});
