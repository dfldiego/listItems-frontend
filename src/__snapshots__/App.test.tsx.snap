// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App> <App /> render correctly 1`] = `
<div>
  <div>
    Loading...
  </div>
</div>
`;
